man/tpm2_getekcertificate.1.md

```diff
@@ -2,32 +2,49 @@
 
 # NAME
 
-**tpm2_getekcertificate**(1) - Retrieve the Endorsement key Certificate for the
-TPM endorsement key from the TPM manufacturer's endorsement certificate hosting
-server.
+**tpm2_getekcertificate**(1) - Retrieve the Endorsement key Certificate.
 
 # SYNOPSIS
 
 **tpm2_getekcertificate** [*OPTIONS*] [*ARGUMENT*]
 
 # DESCRIPTION
 
-**tpm2_getekcertificate**(1) - Retrieve the Endorsement key Certificate for
-the TPM endorsement key from the TPM manufacturer's endorsement certificate
-hosting server. The argument specifies the URL address for the ek certificate
-portal.
+**tpm2_getekcertificate**(1) - Retrieve the endorsement key certificate. The
+certificate is present either on the TCG specified TPM NV indices OR on the TPM
+manufacturer's endorsement certificate hosting server. The certificate lookup is
+in the following order:
+1. Attempts to retrieve from the TPM NV indices first : if an argument
+   specifying the URL address for the EK certificate portal is **not supplied**.
+2. Retrieved from the Intel EK certificate backend:<br>
```

Comment:
you can get rid of the <br> and just add a blank line between this and item "a".